driver-core/src/main/com/mongodb/internal/operation/OperationHelper.java

```diff
@@ -171,6 +171,27 @@ static void validateHint(final ConnectionDescription connectionDescription, fina
         }
     }
 
+    static void validateAllowDiskUse(final Connection connection, final Boolean allowDiskUse) {
+        validateAllowDiskUse(connection.getDescription(), allowDiskUse);
+    }
+
+    static void validateAllowDiskUse(final ConnectionDescription connectionDescription, final Boolean allowDiskUse) {
```

Comment:
This method could be inlined.  It's only called from one place, and inlining it would make the two remaining `validateAllowDiskUse` methods look more alike

Reply:
I have inlined the method.